stdlib/builtin/exception.rbs

```diff
@@ -124,7 +124,7 @@
 class Exception < Object
   def self.to_tty?: () -> bool
 
-  def self.exception: (?String msg) -> self
+  def self.exception: (?String msg) -> Exception
```

Comment:
I think `self` means `Exception` class itself so the test should be failed...?

Reply:
Right, `self` means `singleton(::Exception)` here, and your correction makes sense!